bhohnco/TheWormhole
# Feature/readme

### Is this a fix or a feature?
- Feature
### What is the change or fix?
-Add most of the structure of every layer of a thorough readme
-Has our deployed site
-Links to further education on our stretch tech
-Links to our own bios
### How should this be tested?
-Open and verify no images are causing errors and that deployed site works on all screens
### Linked Issues:
- #36 
#### Add A Reviewer if Necessary
